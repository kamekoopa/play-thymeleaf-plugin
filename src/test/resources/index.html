<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:th="http://www.thymeleaf.org" lang="ja">
<head>
<title>テスト</title>
</head>
<body>
	<div>
		<span th:text="${message}">テスト</span>
	</div>
</body>
</html>
