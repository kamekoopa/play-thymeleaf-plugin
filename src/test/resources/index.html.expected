<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:th="http://www.thymeleaf.org" lang="ja">
<head>
<title>テスト</title>
</head>
<body>
	<div>
		<span>Hello</span>
	</div>
</body>
</html>
